gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint-groups/llm/api-endpoint-groups-llm.component.html

```diff
@@ -0,0 +1,106 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (providersTableData().length > 0) {
+  @for (provider of providersTableData(); track provider.name; let i = $index) {
+    <mat-card class="endpoint-group-card">
+      <mat-card-content>
+        <div class="endpoint-group-card__header">
+          <div class="endpoint-group-card__header__title">
+            <h3 class="mat-h3">{{ provider?.name }}</h3>
+            <span class="gio-badge-primary">
+              {{ getProviderTypeDisplayName(provider?.providerConfiguration?.provider) }}
+            </span>
+          </div>
+          <div class="endpoint-group-card__header__actions">
+            <button
+              *gioPermission="{ anyOf: ['api-definition-u'] }"
+              mat-stroked-button
+              [attr.aria-label]="isReadOnly() ? 'View provider' : 'Edit Provider'"
+              [routerLink]="['./provider', i]"
+            >
+              <mat-icon [svgIcon]="isReadOnly() ? 'gio:eye-empty' : 'gio:edit-pencil'"></mat-icon>
+              {{ isReadOnly() ? 'View' : 'Edit' }}
+            </button>
+            <ng-container *gioPermission="{ anyOf: ['api-definition-u'] }">
+              @if (providersTableData().length > 1) {
+                <button mat-stroked-button aria-label="Delete provider" (click)="deleteGroup(provider.name)" [disabled]="isReadOnly()">
+                  <mat-icon svgIcon="gio:trash"></mat-icon>
+                  Delete
+                </button>
+              }
+            </ng-container>
+          </div>
+        </div>
+        <table
+          mat-table
+          [dataSource]="provider.providerConfiguration.models"
+          [attr.id]="'groupsTable-' + i"
+          aria-label="Endpoint group table"
+          class="gio-table-light"
+        >
+          <!-- Model Column -->
+          <ng-container matColumnDef="model">
+            <th mat-header-cell *matHeaderCellDef id="model">Model</th>
+            <td mat-cell *matCellDef="let element">
+              {{ element.name }}
+            </td>
+          </ng-container>
+
+          <!-- Options Column -->
+          <ng-container matColumnDef="costInput">
+            <th mat-header-cell *matHeaderCellDef id="costInput">Cost per 1m tokens (input) ($)</th>
+            <td mat-cell *matCellDef="let element">
+              {{ element.inputPrice }}
```

Comment:
if it’s a number you can format it like that


```suggestion
              {{ element.inputPrice  | number:'1.2' }}
```